BootloaderCorePkg/Library/SmbiosInitLib/SmbiosTables.h

```diff
@@ -0,0 +1,199 @@
+/** @file
+
+  Copyright (c) 2019, Intel Corporation. All rights reserved.<BR>
+  This program and the accompanying materials
+  are licensed and made available under the terms and conditions of the BSD License
+  which accompanies this distribution.  The full text of the license may be found at
+  http://opensource.org/licenses/bsd-license.php.
+
+  THE PROGRAM IS DISTRIBUTED UNDER THE BSD LICENSE ON AN "AS IS" BASIS,
+  WITHOUT WARRANTIES OR REPRESENTATIONS OF ANY KIND, EITHER EXPRESS OR IMPLIED.
+
+**/
+
+#ifndef __SMBIOS_TABLES_H__
+#define __SMBIOS_TABLES_H__
+
+#include <Library/BaseLib.h>
+#include <IndustryStandard/SmBios.h>
+
+#define TO_BE_FILLED                                0
+
+#define SMBIOS_MISC_STRING_NULL                     0
+#define SMBIOS_MISC_STRING_1                        1
+#define SMBIOS_MISC_STRING_2                        2
+#define SMBIOS_MISC_STRING_3                        3
+#define SMBIOS_MISC_STRING_4                        4
+#define SMBIOS_MISC_STRING_5                        5
+#define SMBIOS_MISC_STRING_6                        6
+#define SMBIOS_MISC_STRING_7                        7
+#define SMBIOS_MISC_STRING_8                        8
+#define SMBIOS_MISC_STRING_9                        9
+
+#define SMBIOS_MISC_UNKNOWN                         "UNKNOWN"
+
+///
+/// SMBIOS_TABLE_TYPE00 Default Strings
+///
+///
+#define SMBIOS_MISC_TYPE00_VENDOR                   L"Intel Corporation"
+#define SMBIOS_MISC_TYPE00_BIOS_VERSION             L"CFL_S/CFL_H/WHL_U SBL"
+#define SMBIOS_MISC_TYPE00_BIOS_RELEASE_DATE        L"04/11/2019"
+
+SMBIOS_TABLE_TYPE0    mBiosInfo = {
+  {                                         // Hdr
+    SMBIOS_TYPE_BIOS_INFORMATION,           /// Hdr.Type
+    sizeof (SMBIOS_TABLE_TYPE0),            /// Hdr.Length
+    0                                       /// Hdr.Handle
+  },
+  SMBIOS_MISC_STRING_1,                     // Vendor
+  SMBIOS_MISC_STRING_2,                     // BiosVersion
```

Comment:
is it a string index or string here? if string, should it be SMBIOS_MISC_TYPE00_BIOS_VERSION?  I don't see SMBIOS_MISC_TYPE00_BIOS_VERSION is used in this patch.

Reply:
It is string index here. I removed those unused macros.